source/features/format-conversation-titles.tsx

```diff
@@ -29,10 +18,8 @@ void features.add(__filebasename, {
 		pageDetect.isPR,
 		pageDetect.isIssue
 	],
-	init() {
-		observeElement(select('#partial-discussion-header')!.parentElement!, init, {
-			subtree: true,
-			childList: true
-		});
```

Comment:
https://github.com/sindresorhus/refined-github/pull/3479 added `subtree` I don't see why it it needed

Reply:
Likely for the same reason why `format-conversation-titles` handled parsed the backticks autonomously instead of using the `parse-backticks` feature: The title is changed by changing a single text node, which isn't a direct child of `select('#partial-discussion-header')!.parentElement!`

Reply:
But git hub refreshes' the entire parent element on change. We are using it as a call back. I don't see why subtree is needed.

Reply:
If that's the case now, it must have changed: https://github.com/sindresorhus/refined-github/pull/3410#issuecomment-665104765

I still see that exact piece of code though:

<img width="890" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/111894099-91a8ec80-89cd-11eb-855b-a966d07dbbf3.png">

Reply:
Is it possible that it only triggers when the title is not markdown? `textContent` would seem to suggest that.

Reply:
I tried almost every event https://github.com/yakov116/pr-deployments/pull/1 and I cannot get it not to work

Reply:
Alright let's assume that they changed how the title updates, since https://github.com/sindresorhus/refined-github/issues/4129#issuecomment-802688144, and that this code path is just not activated (it has an existence check in the received JSON)